stl/inc/chrono

```diff
@@ -2128,8 +2128,11 @@ namespace chrono {
     }
 
     template <class _Duration>
-    requires _Is_duration_v<_Duration> class hh_mm_ss {
+    class hh_mm_ss {
     public:
+        static_assert(_Is_duration_v<_Duration>,
+            "N4885 [time.hms.overview]/2 requires Duration to be a specialization of chrono::duration.");

```

Comment:
Is requires a constraint? Could we add "Mandates:" like below

Reply:
"requires" is old-timey wording for mandates (not constraints, which were specified by "shall not participate in overload resolution unless" :vomiting_face:). The Standardese here says "If `Duration` is not a specialization of `duration`, the program is ill-formed." but I think we can describe this as "mandates" when paraphrasing.